Makefile

```diff
@@ -140,21 +140,21 @@ test-image-all:
 # -----------
 
 .PHONY: test-e2e
-test-e2e: test-e2e-1.24
+test-e2e: test-e2e-1.25
 
 .PHONY: test-e2e-all
-test-e2e-all: test-e2e-1.24 test-e2e-1.23 test-e2e-1.22
+test-e2e-all: test-e2e-1.25 test-e2e-1.24 test-e2e-1.23
```

Comment:
we shouldn't support new versions of Kubernetes in a patch release since it is frozen in time.

Reply:
I have a question, does our v0.6 not need to support kubernetesv1.25?
Might need an update in the readme

Reply:
Oh, indeed I forgot that metrics-server advertises quite a large compatibility with Kubernetes since it doesn't introduce breaking changes often. Ideally, I think we should only advertise what is being tested rather than everything that we think is compatible since this is what we built confidence toward and what we can easily verify.

As such I would personally be in favor of updating the documentation to reflect what is being tested rather than what we think is compatible. Any thoughts @serathius?